features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/InfoBar.jsx

```diff
@@ -295,7 +295,7 @@ class InfoBar extends React.Component {
                 if (name === 'all'){
                     response = 'All Applicable';
                 } else {
-                    response = keyManagers;
+                    response = keyManagers.join();
```

Comment:
This could change as below for simplified ad performance-optimized manner

```
const isAll = keyManagers.find(name => name === 'all');
response = isAll ? 'All Applicable' : keyManagers.join();
```